app/LoginPage/page.js

```diff
@@ -0,0 +1,82 @@
+"use client";
+import React, { useState } from "react";
+import { FiShoppingCart } from "react-icons/fi";
+import { FcGoogle } from "react-icons/fc";
+import { createUserWithEmailAndPassword, signInWithPopup } from "firebase/auth";
+import { auth, provider } from "../../FirebaseConfig/Firebase";
+
+function Login() {
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+  // REDIRECT USER AFTER SIGNIN
+  const RedirectUser = (e) => {
+    window.location.replace("ProductPage");
+  };
+
+  // LOGIN USER WITH EMAIL AND PASSWORD
+  const LoginWithEmail = async (e) => {
+    try {
+      e.preventDefault();
+      await createUserWithEmailAndPassword(auth, email, password);
+      RedirectUser();
+    } catch (error) {
+      console.log(error);
+    }
+  };
+  // LOGIN USER WITH GOOGLE MAIL
+  const LoginWithGoogle = async (e) => {
+    try {
+      e.preventDefault();
+      await signInWithPopup(auth, provider);
+      console.log(auth.currentUser, auth.currentUser.photoURL);
+      RedirectUser();
+    } catch (error) {
+      console.log(error);
+    }
+  };
+
+  return (
+    <div className="flex justify-center my-20  ">
+      <div className="bg-cyan-900 w-1/3 h-96 p-24 shadow-lg text-center">
+        <FiShoppingCart className="m-auto text-white text-8xl" />
+        <span className=" text-white text-2xl"> Super Shopping</span>
+      </div>
+
+      <div className="w-1/3 h-96 bg-yellow-100 shadow-lg">
+        <form className="pl-20 pt-8">
```

Comment:
Oh, I see why you saved the email and password.

You don't have to do that though, you can easily access the info when the form is submitted.

See this as a guide:  https://contactmentor.com/login-form-react-js-code/